app/views/devise/registrations/new.html.erb

```diff
@@ -1,27 +1,32 @@
-<h2>Sign up</h2>
+<div class="position-relative">
+    <div class="position-absolute top-0 start-50 translate-middle-x">
```

Comment:
code formatting  looks weird - same thing applies to other devise views

Reply:
Okay, I will fix that.